docs/samples/contact-center/app.js

```
@@ -1095,6 +1082,10 @@ function doAgentLogin() {
     
   }).catch((error) => {
     console.log('Agent Login failed', error);
+    if(error.moreDetails.fieldName = 'input')  {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)